revolv/payments/tests/models/test_payment_instrument_type.py

```diff
@@ -1,12 +1,16 @@
 from django.test import TestCase
-from revolv.payments.models import INSTRUMENT_PAYPAL, PaymentInstrumentType
+from revolv.payments.models import (INSTRUMENT_CHECK, INSTRUMENT_PAYPAL,
+                                    INSTRUMENT_REINVESTMENT,
+                                    INSTRUMENT_REPAYMENT,
+                                    PaymentInstrumentType)
 
 
 class PaymentInstrumentTypeTest(TestCase):
 
-    fixtures = ['supported_instruments.json']
-
     def test_get_payment_instrument_types(self):
 
-        self.assertEqual(1, len(PaymentInstrumentType.objects.all()))
+        self.assertEqual(4, len(PaymentInstrumentType.objects.all()))
         self.assertEqual(INSTRUMENT_PAYPAL, PaymentInstrumentType.objects.first().name)
+        self.assertEqual(INSTRUMENT_REINVESTMENT, PaymentInstrumentType.objects.get(id=2).name)
```

Comment:
I think there was a `get_paypal()` method around here somewhere that I wrote for this reason: we can go off that with `get_reinvestment()` and `get_repayment()`